roles/openshift_master_upgrade/handlers/main.yml

```diff
@@ -0,0 +1,35 @@
+---
```

Comment:
If we didn't have the role dependencies to worry about, I would say that we could just add a tasks/noop.yml file to the role, then you could use
```
- include_role:
    tasks_from: noop
```

If we had a shared common role that housed handlers, variables, default values, etc but didn't have any role dependencies, then we could pull that in instead.

That was kind of the goal of roles like openshift_master_facts, but it has the nasty dependency on openshift_facts currently.